rpcserver.go

```diff
@@ -1437,8 +1449,8 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 		paymentMap := map[string]int64{targetAddr.String(): in.Amount}
 		err := wallet.WithCoinSelectLock(func() error {
 			newTXID, err := r.sendCoinsOnChain(
-				paymentMap, feePerKw, minConfs,
-				label, coinSelectionStrategy,
+				paymentMap, feePerKw, minConfs, label,
+				coinSelectionStrategy, selectOutpoints,
```

Comment:
Didn't we want to set a manual coin selection strategy further above in `lnrpc.UnmarshallCoinSelectionStrategy` if utxos are selected?

Reply:
No , we just changed the `inputSource` in btcwallet for select utxos only, https://github.com/Chinwendu20/btcwallet/blob/d75f2c3f477affcf636542e2ec1538ffcddf4bae/wallet/createtx.go#L201
entirely skipping arranging coins if we have the utxos that we want to use already.